pvlib/tests/test_shading.py

```diff
@@ -104,3 +107,155 @@ def test_sky_diffuse_passias_scalar(average_masking_angle, shading_loss):
     for angle, loss in zip(average_masking_angle, shading_loss):
         actual_loss = shading.sky_diffuse_passias(angle)
         assert np.isclose(loss, actual_loss)
+
+
+@pytest.fixture
+def true_tracking_angle_and_inputs_NREL():
```

Comment:
This test is good for normal inputs.  I think it would be good to complement it with another test that checks edge cases.  Here are some test edge cases that I came up with (please verify that they make sense!):

```python
df = pd.DataFrame(
    data=[
     [0, 0, 0, 0, 0],
     [0, 180, 0, 0, 0],
     [0, 0, 0, 180, 0],
     [0, 180, 0, 180, 0],
     [45, 0, 0, 180, 0],
     [45, 90, 0, 180, -45],
     [45, 270, 0, 180, 45],
     [45, 90, 90, 180, -90],
     [45, 270, 90, 180, 90],
     [45, 90, 90, 0, 90],
     [45, 270, 90, 0, -90],
     [45, 45, 90, 180, -135],
     [45, 315, 90, 180, 135]],
    columns=['solar_zenith', 'solar_azimuth', 'axis_tilt', 'axis_azimuth', 'psza']
)
```